setup.cfg

```diff
@@ -44,12 +44,12 @@ dev =
     bumpver>=2023.1129
     pgtest~=1.3
     pre-commit>=3.5
-    pytest~=8.2.0
+    pytest~=8.3.0
     pytest-cov~=5.0
-    pytest-docker~=3.0
-    pytest-selenium~=4.1
-    pytest-timeout~=2.2
-    selenium==4.20.0
+    pytest-docker~=3.1.0
+    pytest-selenium~=4.1.0
+    pytest-timeout~=2.3.0
```

Comment:
To be on the safe side and to reduce potential churn, let's pin all test dependencies to minor versions.